### ISSUE TITLE
The effect of the `base_acc` in accuracy_predictor ?

### ISSUE THREAD

**Issue by LicharYuan** [mit-han-lab/once-for-all#42]
Hi, I noticed that here(link1) add base acc to train accuracy predictor and set the last linear layer bias as False. 
For the same batch data, they have the same bias. 
If I do not add this bias, is the predictor will decrease?  Or I missing any analysis in the paper?

link1 : https://github.com/mit-han-lab/once-for-all/blob/master/ofa/nas/accuracy_predictor/acc_predictor.py#L32

**Comment by Fr1ck** [comment]
The author has explained at #30.
You can simply replace the zero tensor to a average accuracy that calcuated from your {architecture, accuracy} datasets, which works well during my test.

**Comment by kourner** [comment]
> 

I have no idea how to train the accuracy predictor, can you share your training script to me? Thanks a million.

**Comment by Fr1ck** [comment]
I followed the author's explanation at #30  and replaced the zero tensor with average accuracy. However, the predictor still can not converge well T_T

**Comment by kourner** [comment]
> I followed the author's explanation at #30 and replaced the zero tensor with average accuracy. However, the predictor still can not converge well T_T
我打算做一个ofa的detection版本，但目前卡在训练acc_pridictor了。如果你也感兴趣的话，方便留下你的联系方式吗？
